client/examples/tutorial.rs

```diff
@@ -185,3 +187,55 @@ fn asset_registration_test(config: &Configuration) -> Result<(), Error> {
     // Finish the test successfully
     Ok(())
 }
+
+fn asset_minting_test(config: &Configuration) -> Result<(), Error> {
+    // BEGIN FRAGMENT: mint_asset
+    use std::str::FromStr;
+
+    use iroha_client::client::Client;
+    use iroha_data_model::{
+        prelude::{AccountId, AssetDefinitionId, AssetId, MintBox},
+        IdBox, Value,
+    };
+    // Create an Iroha client
+    let iroha_client: Client = Client::new(&config).unwrap();
+    // Define the instances of an Asset and Account
+    let roses = AssetDefinitionId::from_str("rose#wonderland")
+        .expect("Valid, because the string contains no whitespace, has a single '#' character and is not empty after");
+    let alice: AccountId = "alice@wonderland".parse()
+        .expect("Valid, because the string contains no whitespace, has a single '@' character and is not empty after");
+    // Mint the Asset instance
+    let mint_roses = MintBox::new(Value::U32(42), IdBox::AssetId(AssetId::new(roses, alice)));
```

Comment:
Add section with alternative syntax: 
```rust
let mint_roses_again = MintBox::new(Value::U32(24), IdBox::AssetId("rose##alice@wonderland".parse()?));
// The syntax is `asset_name#asset_domain#account_name@account_domain`, or `roses.to_string() + "#" + alice.to_string()`.
// The `##` is a short-hand for the rose `which belongs to the same domain as the account to which it belongs to. 
```

Reply:
You could do this in the `Burn` test, though.

Reply:
Understood, will apply to both in the morning!